arm-billing/2017-02-27-preview/swagger/billing.json

```diff
@@ -0,0 +1,308 @@
+        "tags": [
+          "Invoices"
+        ],
+        "operationId": "Invoices_GetLatest",
+        "description": "Gets the most recent invoice. When getting a single invoice, the downloadUrl property is expanded automatically.",
+        "x-ms-examples": {
+					"InvoicesGetLatest": {
+    				"$ref": "../examples/InvoicesGetLatest.json" 
+        	}
+        },
+        "parameters": [
+          {                                 
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+                "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Invoice": {
+      "description": "An invoice resource can be used download a PDF version of an invoice.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/InvoiceProperties"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "DownloadUrl": {
+      "description": "Secure URL that can be used temporarily to download a PDF invoice. May be null if returned from a list operation.",
+      "properties": {
+        "expiryTime": {
+          "description": "The time in UTC at which this download link will expire.",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "url": {
+            "description": "The URL to the PDF file.",
+            "readOnly": true,
+            "type": "string"
+        }
+      }
+    },
+    "InvoiceProperties":{
+      "description": "The properties of the invoice.",
+      "properties": {
+        "invoicePeriodStartDate": {
+          "description": "The start of the date range covered by the invoice.",
+          "readOnly": true,
+          "type": "string",
+          "format": "date"
+        },
+        "invoicePeriodEndDate": {
+          "description": "The end of the date range covered by the invoice.",
+          "readOnly": true,
+          "type": "string",
+          "format": "date"
+        },
+        "downloadUrl": {
+          "$ref": "#/definitions/DownloadUrl"
```

Comment:
Note that we allow overriding the model type's description here (maybe this is helpful to you regarding my comment on `DownloadUrl`s description). Just put another `description` right next to this `$ref`